unicorn/src/module.cpp

```diff
@@ -25,6 +26,7 @@ validation() -> api::preconditions_t {
 void
 initialize(api::repository_t& repository) {
     repository.insert<service::unicorn_service_t>("unicorn");
+    repository.insert<cluster::unicorn_cluster_t>("unicorn_cluster");
```

Comment:
These are different categories, so names won't clash. Make it "unicorn" in both cases.